src/install/PackageManager/CommandLineArguments.zig

```diff
@@ -50,7 +50,7 @@ const shared_params = [_]ParamType{
     clap.parseParam("--omit <dev|optional|peer>...         Exclude 'dev', 'optional', or 'peer' dependencies from install") catch unreachable,
     clap.parseParam("--lockfile-only                       Generate a lockfile without installing dependencies") catch unreachable,
     clap.parseParam("--linker <STR>                        Linker strategy (one of \"isolated\" or \"hoisted\")") catch unreachable,
-    clap.parseParam("--minimum-release-age <NUM>           Only install packages published at least N seconds ago (security feature)") catch unreachable,
+    clap.parseParam("--minimum-release-age <STR>           Only resolve package versions that are at least this old (e.g., 3d, 24h)") catch unreachable,
```

Comment:
This is likely to break old uses. Can we make it so that this supports both a string and a number?

Reply:
That's just the type shown, it doesn't have any real significance. For context the actual part in clde where I handle it, I try to parse as number and then parse `ms` and then error if still can't work.